beamer/tests/agent/multiple_processors/test_multiple_event_processors.py

```diff
@@ -0,0 +1,241 @@
+import contextlib
+import json
+import os
+import signal
+import time
+from pathlib import Path
+from subprocess import DEVNULL, PIPE
+from typing import Any
+
+import ape
+import eth_account
+import psutil
+import yaml
+from eth_utils import to_canonical_address
+from yaml.loader import SafeLoader
+
+import beamer.agent.metrics
+from beamer.agent.agent import Agent
+from beamer.agent.config import Config
+from beamer.agent.contracts import ContractInfo
+from beamer.agent.typing import BlockNumber, ChainId
+from beamer.agent.util import TokenChecker, make_web3, transact
+from beamer.tests.util import Sleeper
+
+SLAVE_TEST_PATH = Path(__file__).parent / "test_slave.py"
+CONFIG_PATH = os.path.join(os.getcwd(), "ape-config.yaml")
+BACKUP_PATH = os.path.join(os.getcwd(), "ape-config.yaml.2")
```

Comment:
Why do we assume `ape-config.yaml` will be located at `os.getcwd()`?

Reply:
because it is :) getcwd() basically the project root as we can say and ape-config.yaml is there.